docs/current-1.1/contributing.md

```diff
@@ -0,0 +1,53 @@
+# Using the current-1.1 folder and sub-folders
```

Comment:
I don't have a strong preference, but I was just wondering if the folder name should have core like core-current or something like that, to make it clear the content in that folder is for .net core.